### ISSUE TITLE
Death touch countdown

### ISSUE THREAD

**Issue by Gogo135** [smasherprog/EqTool#141]
Love the application. Is it possible to add a Death Touch countdown overlay with adjustable countdown time?

Thank you.

**Comment by smasherprog** [comment]
yeah i have an update that im working on which will expand alot of functionality
Its a large code update so its going to be a little bit
